components/deprecated-en.html

```diff
@@ -24,7 +24,7 @@ <h2>Arm's length branding - Markup change - v4.0.28</h2>
 <aside class="gc-nttvs container">
 	<div class="row mrgn-tp-lg">
 		<div class="col-sm-4 col-xs-10 col-sm-push-8 mrgn-bttm-lg">
-			<img class="img-responsive" alt="Department name" src="{{assets}}/../{{site.theme}}/assets/sig-blk-en.png">
+			<img class="img-responsive" alt="Department name" src="../templates/institutional/img/arms-en.png">
```

Comment:
@duboisp FYI there is one instance of a /components/ referencing an image from /templates/... Although it's the deprecated page talking about arm's length branding (which is technically supposed to be specific to the arm's length template).

Apart from that, there are no other instances of component pages referencing template images.

Reply:
Yes, that page need to be dismantled and get it content moved into where the belong, like that deprecated example need to be moved into the corresponding template.